src/components/input/DurationInput.js

```diff
@@ -13,9 +13,15 @@ const DurationInput = ({ value, onChange, onBlur, onFocus, onKeyDown, autoFocus,
     }
 
     if (hours === '') {
+      if (typeof value?.minutes === 'undefined') {
+        onChange();
```

Comment:
Can you explain why it's triggering an onChange with no params? Don't really understand what's going on here.

Reply:
Here's the different states of the value:

```js
// No values in either hours or minutes
value: undefined

// No value in hours
value: {
  minutes: 30
}

// No value in minutes
value: {
  hours: 1
}

// Value in both minutes and hours
value: {
  hours: 1
  minutes: 30
}
```

Because each input can be cleared independently, we want to pass no object at all if each input is empty